app/controllers/admin/infobase_controller.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Admin
+  class InfobaseController < ApplicationController
+    def index
+      @users = User.all
+      @menus = Menu.all
+      @menu = Menu.new(params.permit(:data))
+      if @menu.save
+        redirect_to admin_infobase_index_path
+      else
+        redirect_to admin_infobase_index_path
+      end
+      @cart_items = CartItem.all
+      @carts = Cart.all
```

Comment:
a lot of instance variable in an action is a no-go
please read this: https://medium.com/kkempin/facade-design-pattern-in-ruby-on-rails-710aa88326f
facades are the best for this purposes